### DISCUSSION TITLE
It's hard to find failing checks

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#108342] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

https://github.com/decidim/decidim/pull/12446#pullrequestreview-1878554899

<img width="920" alt="image" src="https://github.com/community/community/assets/2119212/b39f5826-be2e-4aa3-b049-635f642b2919">

> Some checks were not successful
> 100 successful and 1 failing checks

Scrolling through 101 checks trying to find the 1 check that failed is really painful

Notes:
* Searching for `❌` doesn't work

* It'd be nice if I could:
  1. sort by status
  2. filter by status

**Comment by Ariffazeel99** [comment]
You need to have at least 2 approvals, and for this merge request, you have only 1 approval at this time. 
so you can solve the issue by having 1 more approval.
